core/new-gui/src/app/dashboard/service/workflow-executions/workflow-executions.service.ts

```diff
@@ -34,4 +35,11 @@ export class WorkflowExecutionsService {
       eId,
     });
   }
+
+  updateWorkflowExecutionsName(wid: number | undefined, eid: number, executionName: string): Observable<Response> {
+    return this.http.post<Response>(
+      `${WORKFLOW_EXECUTIONS_API_BASE_URL}/update_name_${wid}_${eid}_${executionName}`,
+      null
```

Comment:
since the method is `post` then send the params in the body instead of null.

Reply:
API has been changed to "update_execution_name" and parameters are transferred in body